### ISSUE TITLE
TLS 1.2

### ISSUE THREAD

**Issue by cjsinnbeck** [zendesk/zendesk_api_client_php#395]
Not so much an issue, but a question: Zendesk is ending support for TLS 1.0 and TLS 1.1 in three days (June 28th). On this page

https://support.zendesk.com/hc/en-us/articles/360000710347

the following is stated:

"Any web application calling Zendesk APIs and attempting to make a connection using TLS 1.0 or 1.1 will no longer work"

The question: Will the Zendesk API PHP Client automatically try/force to connect with TLS 1.2, or must this be added as a parameter somewhere? I did a quick code search but cannot find any place where Guzzle is set up to use TLS 1.2

Best regards,
Christoffer

**Comment by joseconsador** [comment]
Hi @cjsinnbeck , I think this [stackoverflow post](https://stackoverflow.com/questions/35512458/forcing-tls-1-2-in-guzzle) answers your question.

tldr;

> If you are certain that:
> 
> -  The underlying server has cURL/libcurl version 7.35.0 or newer installed…
> -  You have a new enough version of PHP installed (recommended at least 5.6.3)…
> -  And are using Guzzle for 100% of your calls to HTTP(S) endpoints…
>
> Then you are in good shape. Once the service flips the switch to require TLS 1.2, your application will automatically start using TLS 1.2.

**Comment by cjsinnbeck** [comment]
@joseconsador Thanks a lot for following up. I have verified that the server and my code meets the criteria in the Stack Overflow post.

Also, I have verified with Zendesk support that our instance of Zendesk has actually been migrated to TLS 1.2, and it has. So since everything is still working, I guess everything... still works :-) In other words, I don't need to change anything here and now.

Best regards Chris
